plugin.video.twitch/resources/lib/converter.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+import xbmc
+import xbmcgui
+from twitch.constants import Keys
+from constants import Images
+from utils import theArt, TitleBuilder
+
+
+class PlaylistConverter(object):
+    @staticmethod
+    def convertToXBMCPlaylist(InputPlaylist, title='', image=''):
+        # Create playlist in Kodi, return tuple (playlist, {listitem dict}) or ()
+        playlist = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
+        playlist.clear()
+        initialItem = None
+        for (url, details) in InputPlaylist:
+            if url:
+                if details != ():
+                    (title, image) = details
+                playbackItem = xbmcgui.ListItem(label=title, thumbnailImage=image, path=url)
```

Comment:
the thumbnailImage init field is now deprecated. Please change to listItem.setArt() in your next update.